src/app/ApplicationTemplate.Presentation/Framework/Version/IVersionProvider.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace ApplicationTemplate;
+
+public interface IVersionProvider
+{
+	/// <summary>
+	/// Gets the application version build.
+	/// </summary>
+	/// <returns>The application version build.</returns>
+	public string Build { get; }
+
+	/// <summary>
+	/// Gets the application version (Major.Minor.Build.Revision).
+	/// </summary>
+	/// <returns>The application version.</returns>
+	public string Version { get; }
```

Comment:
Couldn't we use the type `Version` instead of string?

Reply:
Yes.